presto-hive-metastore/src/main/java/com/facebook/presto/hive/HiveStorageFormat.java

```diff
@@ -161,7 +166,7 @@ public boolean equals(Object o)
                 return false;
             }
             SerdeAndInputFormat that = (SerdeAndInputFormat) o;
-            return serDe.equals(that.serDe) && inputFormat.equals(that.inputFormat);
+            return Objects.equals(serDe, that.serDe) && Objects.equals(inputFormat, that.inputFormat);
```

Comment:
This is definitely the right thing to do, but did you encounter any issues with older logic?

Reply:
Yes, the TestTableMetadata was failing with NPE in the method after adding the new format.

```
    @Test
    public void testAssertTableMetadataEquals()
    {
        assertTableMetadataEquals(createTableMetadata(null), createTableMetadata(null));
    }
```